base/set.jl

```diff
@@ -104,7 +104,7 @@ See also: [`in`](@ref), [`push!`](@ref), [`Set`](@ref)
     This function requires at least 1.11.
 
 # Examples
-```jldoctest; filter = r"^  [1234]\$"
+```jldoctest; filter = r"(?m)^\\s+\\S"
```

Comment:
This is not a good filter. It will remove all the outputs from the doctests, where what we want is only to make sure that the integers in the final example can occur in any order.

Reply:
I don't think it will remove all the outputs, right? I intended this to match `(at least one whitespace char) and then (exactly one non-whitespace char)`, which in this instance will match the elements ` 4\n 2\n 3\n 1` after `julia> s` but none of the others

maybe `r"(?m)^\s+[1234]"` would be a bit more specific? at least then we're testing for the right elements. it won't catch if the set returns `1, 1, 1, 1` but also I think in general that's something a unit test needs to cover; doctests cannot be so comprehensive.